src/distilabel/steps/tasks/benchmarks/arena_hard.py

```diff
@@ -0,0 +1,59 @@
+    @property
+    def inputs(self) -> List[str]:
+        return ["instruction", "generations"]
+
+    def format_input(self, input: StepInput) -> ChatType:
+        return [
+            {
+                "role": "system",
+                "content": "Please act as an impartial judge and evaluate the quality of the responses provided by two AI assistants to the user prompt displayed below. You will be given assistant A's answer and assistant B's answer. Your job is to evaluate which assistant's answer is better.\n\nBegin your evaluation by generating your own answer to the prompt. You must provide your answers before judging any answers.\n\nWhen evaluating the assistants' answers, compare both assistants' answers with your answer. You must identify and correct any mistakes or inaccurate information.\n\nThen consider if the assistant's answers are helpful, relevant, and concise. Helpful means the answer correctly responds to the prompt or follows the instructions. Note when user prompt has any ambiguity or more than one interpretation, it is more helpful and appropriate to ask for clarifications or more information from the user than providing an answer based on assumptions. Relevant means all parts of the response closely connect or are appropriate to what is being asked. Concise means the response is clear and not verbose or excessive.\n\nThen consider the creativity and novelty of the assistant's answers when needed. Finally, identify any missing important information in the assistants' answers that would be beneficial to include when responding to the user prompt.\n\nAfter providing your explanation, you must output only one of the following choices as your final verdict with a label:\n\n1. Assistant A is significantly better: [[A>>B]]\n2. Assistant A is slightly better: [[A>B]]\n3. Tie, relatively the same: [[A=B]]\n4. Assistant B is slightly better: [[B>A]]\n5. Assistant B is significantly better: [[B>>A]]\n\nExample output: \"My final verdict is tie: [[A=B]]\".",
+            },
+            {
+                "role": "user",
+                "content": f"<|User Prompt|>\n{input['instruction']}\n\n<|The Start of Assistant A's Answer|>\n{input['generations'][0]}\n<|The End of Assistant A's Answer|>\n\n<|The Start of Assistant B's Answer|>\n{input['generations'][1]}\n<|The End of Assistant B's Answer|>",
+            },
+        ]
+
+    @property
+    def outputs(self) -> List[str]:
+        return ["evaluation", "result", "model_name"]
+
+    def format_output(
+        self, output: Union[str, None], input: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        if output is None:
+            return {"evaluation": None, "result": None}
+        pattern = re.compile(r"\[\[([AB<>=]+)\]\]")
+        match = pattern.search(output)
+        if match is None:
+            return {"evaluation": output, "result": None}
+        return {"evaluation": output, "result": match.group(1)}
+
+
+class ArenaHardResults(GlobalStep):
```

Comment:
Is this intended to be empty or is it WIP?

Reply:
Yes, it's still WIP since I was double checking that still, I'll push that later today ideally! 👍🏻